starter_code/canvas.js

```diff
@@ -2,15 +2,17 @@
 class HangmanCanvas {
   constructor(secretWord) {
     this.ctx = document.getElementById('hangman').getContext('2d');
-    
   }
 
   createBoard() {
-
+    this.createBoard()
   }
 
   drawLines() {
-
+    this.ctx.beginPath()
+    this.ctx.moveTo(700, 500)
+    this.ctx.lineTo(750, 500)
+    this.ctx.closePath()
```

Comment:
despues de que cierres el path tienes que agregar
` this.ctx.stroke()`para que se pueda pintar.